plugins/services/src/js/containers/tasks/TasksView.js

```diff
@@ -161,6 +162,13 @@ class TasksView extends mixin(SaveStateMixin) {
       return service.getServiceStatus().key === ServiceStatusTypes.DEPLOYING;
     });
 
+    const isSDK = Object.keys(checkedItems).some(function(taskId) {
+      const service = DCOSStore.serviceTree.getServiceFromTaskID(taskId);
+      const isSDK = isSDKService(service);
```

Comment:
#nit but we don't need to assign to a variable here. We can just `return isSDKService(service);`